homework.js

```diff
@@ -0,0 +1,119 @@
+// 1
+function numberSum(n) {
+    let total = 0;
+      for(let i = 1; i <= n; i++){
+        total += i;
+      }
+      return total;
+  }
+
+// 2
+    function numberEven(n){
+      let sum = 0;
+      for(let i = 1; i< n; i++) {

```

Comment:
the condition should be "i<=n", not "n<i". 
"n<i" gives the wrong value for number that are even.